composeml/label_maker.py

```diff
@@ -122,16 +76,39 @@ def search(self, df, minimum_data, num_examples_per_instance, gap, verbose=True,
         df = self._preprocess(df)
 
         assert_valid_offset(minimum_data)
-        assert_valid_offset(self.window_size)
         assert_valid_offset(gap)
 
-        df_to_labels = on_slice(
-            self.labeling_function,
-            min_data=minimum_data,
-            window=self.window_size,
-            n_examples=num_examples_per_instance,
-            gap=gap,
-        )
+        def df_to_labels(df):
+            name = self.labeling_function.__name__
+            labels = pd.Series(name=name)
+
+            df = df.loc[df.index.notnull()]
+            df.sort_index(inplace=True)
+
+            if df.empty:
+                return labels.to_frame()
+
+            cutoff_time = offset_time(df.index, minimum_data)
+
+            for example in range(num_examples_per_instance):
+                df = df[cutoff_time:]
+
+                if df.empty:
+                    break
+
+                time = offset_time(df.index, self.window_size)
+                label = self.labeling_function(df[:time], *args, **kwargs)
+
+                not_none = label is not None
+                not_nan = label is not pd.np.nan
+                if not_none and not_nan:
```

Comment:
i believe you can just use `not pd.isnull(label)`

Reply:
Refactor to use `pd.isnull` in [this](https://github.com/FeatureLabs/compose-ml/pull/26/commits/b438280fdd8ad7d75d32a2a44ee1a86ae8e78de6#diff-22bf41f653d21f0ca3dd15dadf5e01e9R102) commit.